docs/references/vod/upload.mdx

```diff
@@ -73,3 +74,52 @@ curl --location --request PUT "${url}" \
 Notice that we only support mp4 files encoded with H264 video and AAC audio for
 now. If you are unsure about these, try sending the file anyway and check any
 returned errors!
+
+# Resumable Uploads
+
+Livepeer Studio supports resumable uploads via [Tus](https://tus.io/).
+This section provides a simple example of how to use `tus-js-client` to upload a video file.
+
+From the previous section, we generated a URL to upload a video file to Livepeer on `POST /api/asset/request-upload`.
+We can use the `tusEndpoint` field of the response to upload the video file and track the progress:
+
+```jsx
+  const file = fs.createReadStream(path);
+  const upload = new tus.Upload(file, {
+      endpoint: tusEndpoint,
+      urlStorage: new (tus as any).FileUrlStorage("/tmp/file"),
+      metadata: {
+        filename,
+        filetype: "video/mp4",
+      },
+      uploadSize: size,
+      onError(error) {
+        throw error;
+      },
```

Comment:
Either make a `console.error()` from the error or I think we shouldn't suggest this error handler since it just throws (probably the default behavior of the lib anyway).